lua/gatekeeper/dylib.lua

```diff
@@ -171,3 +171,8 @@ function print_lls_dump_entry(lls_dump_entry, acc)
 		", ip: " .. ip .. ", mac: " .. ha ..
 		", port: " .. port_id .. "]\n"
 end
+
+function bytecode_dump(filename)
```

Comment:
1. The parameter of this function should be `lua_code`;
2. This function should be called `update_gt_lua_states_incrementally()` and should call `l_update_gt_lua_states_incrementally()` passing the byte code. This avoids calling Lua from C and avoids having a file to pass the code that will run at the Lua states of the GT blocks.
3. We need an example of calling this function in `lua/examples/example_of_dynamic_config_request.lua`.